MOBILE_API.xml

```diff
@@ -5829,17 +5847,26 @@
         <param name="rpm" type="Boolean" mandatory="false">
             <description>The number of revolutions per minute of the engine</description>
         </param>
-        <param name="fuelLevel" type="Boolean" mandatory="false">
-            <description>The fuel level in the tank (percentage)</description>
+        <param name="fuelLevel" type="Boolean" mandatory="false" deprecated="true" since="6.2">
+            <description>The fuel level in the tank (percentage). This parameter is deprecated starting RPC Spec 6.2, please see fuelRange.</description>
+            <history>
+                <param name="fuelLevel" type="Boolean" mandatory="false" since="2.0" until="6.2"/>
+            </history>
         </param>
-        <param name="fuelLevel_State" type="Boolean" mandatory="false">
-            <description>The fuel level state</description>
+        <param name="fuelLevel_State" type="Boolean" mandatory="false" deprecated="true" since="6.2">
+            <description>The fuel level state. This parameter is deprecated starting RPC Spec 6.2, please see fuelRange.</description>
+            <history>
+                <param name="fuelLevel_State" type="Boolean" mandatory="false" since="2.0" until="6.2"/>
```

Comment:
```suggestion
                <param name="fuelLevel_State" type="Boolean" mandatory="false" since="2.0" until="7.0"/>
```
Please make similar version changes for this parameter in the other vehicleData RPCs.